tests/unit/drivers/test_cache_driver.py

```diff
@@ -146,7 +146,7 @@ def test_cache_content_driver_same_content(tmpdir, test_metas):
     doc1.text = new_string
     doc1.update_content_hash()
     with BaseExecutor.load(filename) as executor:
-        executor.update([UniqueId(1)], [doc1.content_hash])
+        executor.update(['1'], [doc1.content_hash])
```

Comment:
To be consistent with the id setting in generation, this should be `...[1]...`. Or change the indexed Documents to `Document(id='1')`

Reply:
changed